ci/azure-pipelines-docs-integration.yml

```
@@ -403,7 +428,7 @@ stages:
       variables:
         python.version: '3.8'
       services:
-        mysql: mysql
+        trino: trino
```

With the fixes mentioned in [this thread](https://github.com/great-expectations/great_expectations/pull/7735/files#r1177970823) could this change be reverted?
```suggestion
        mysql: mysql
```
Couple other notes:
Will the multiple database tests run with this change? I think they have mysql / postgres (seeing this one for example: https://github.com/great-expectations/great_expectations/blob/develop/tests/integration/test_definitions/multiple_backend/integration_tests.py#L6-L18)

Also the pytest command in this stage is using `--mysql`

---

Maybe instead we also disable this stage if more work is needed to enable mysql (or convert fully to trino). As it stands I think we are just skipping the tests so this stage isn't doing anything.

---

Even with that stuff added, mysql still fails which is why the entire mysql stage is skipped

---

That example you linked looks to be mysql and postgresql in particular. I assumed that the "multiple databases" tests were just be run all the things with whatever 2 backends are passed as `--options` to pytest.

---

The test runner just skips based on which databases are passed as options. I.e. you need to pass the right databases to pytest e.g. `pytest --postgresql --mysql` or `pytest --postgresql --trino` depending on which direction you take (reenable mysql or convert to trino). Maybe better to disable this stage in this PR just so it doesn't look like it is passing.